v1/api/swagger.yaml

```diff
@@ -4650,34 +5298,47 @@ x-stoplight:
         \ instance-shutdown`\n\n\n### Third party\n\n#### Chef cookbooks\n\n[https://github.com/nshenry03/jumpcloud](https://github.com/nshenry03/jumpcloud)\n\
         \n[https://github.com/cjs226/jumpcloud](https://github.com/cjs226/jumpcloud)"
       public: true
-    multi-tenant-organization-api-header:
-      id: "multi-tenant-organization-api-header"
-      name: "Multi Tenant Organization API Header"
-      content: "The Multi-Tenant Organzation API Header is available for JumpCloud\
-        \ Admins to use when making API requests from Organizations that have multiple\
-        \ managed organizations. \n\nThe `x-org-id` is a required header for all multi-tenant\
+    multi-tenant-organization-api-headers:
+      id: "multi-tenant-organization-api-headers"
+      name: "Multi Tenant Organization API Headers"
+      content: "Multi-Tenant Organzation API Headers are available for JumpCloud Admins\
```

Comment:
spelling

Reply:
fixed. stoplight doesn't show typos in the text sections.. so annoying!